core/node/crypto/config.go

```diff
@@ -95,6 +95,8 @@ type OnChainSettings struct {
 	StreamCacheExpiration    time.Duration `mapstructure:"stream.cacheExpirationMs"`
 	StreamCachePollIntterval time.Duration `mapstructure:"stream.cacheExpirationPollIntervalMs"`
 
+	MbsListLimit uint64 `mapstructure:"stream.miniblocksListLimit"`
```

Comment:
Lets name it explicitly by API name:
```suggestion
	GetMiniblocksMaxPageSize uint64 `mapstructure:"stream.getMiniblocksMaxPageSize"`
```